src/web/client/WebExtensionContext.ts

```diff
@@ -387,8 +387,9 @@ class WebExtensionContext implements IWebExtensionContext {
                     new Date().getTime() - requestSentAtTime,
                     response?.statusText,
                     '',
+                    telemetryEventNames.WEB_EXTENSION_API_REQUEST_FAILURE,
+                    response?.status.toString(),
                     telemetryEventNames.WEB_EXTENSION_POPULATE_LANGUAGE_ID_TO_CODE_API_ERROR,
```

Comment:
we should not be using the telemetry event name as method name.

Reply:
fixed